requirements.txt

```diff
@@ -1,6 +1,6 @@
 setuptools >= 21.0.0
 urllib3 >= 1.15.1
 requests ~= 2.31
-wasmtime == 9.0.0
+wasmtime == 23.0.0
```

Comment:
That's a big jump, any API differences?

Reply:
None from the docs - they like to do majors bumps instead of minors.